t/40-script_openqa-clone-custom-git-refspec.t

```diff
@@ -40,12 +40,20 @@ my $args
   = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9128 https://openqa.opensuse.org/tests/1107158';
 isnt run_once($args), 0, 'without network we fail (without error)';
 # mock any external access with all arguments
-$ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my_branch"}}'; true};
+$ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my_branch"}, "body": "Lorem ipsum"}'; true};
 $ENV{curl_openqa}
   = qq{echo -e '{"TEST": "my_test", "CASEDIR": "/my/case/dir", "PRODUCTDIR": "/my/case/dir/product", "NEEDLES_DIR": "/my/case/dir/product/needles"}'; true};
 my $expected
   = qr{openqa-clone-job --skip-chained-deps --within-instance https://openqa.opensuse.org 1107158 _GROUP=0 TEST=my_test\@user/os-autoinst-distri-opensuse#my_branch BUILD=user/os-autoinst-distri-opensuse#9128 CASEDIR=https://github.com/user/os-autoinst-distri-opensuse.git#my_branch PRODUCTDIR=os-autoinst-distri-opensuse/product NEEDLES_DIR=/my/case/dir/product/needles};
-stdout_like sub { $ret = run_once($args); }, $expected, 'clone-job command line is correct';
+combined_like sub { $ret = run_once($args); }, $expected, 'clone-job command with test from command line';
+is $ret, 0, 'command exits successfully';
+
+$ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my_branch"}, "body": "\@openqa: Clone https://openqa.opensuse.org/tests/1107158"}'; true};
+$args
+  = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9128';
```

Comment:
hm, this starts to look ugly. As it is not important if the simulated github repos actually exist, could you use something shorter? E.g. `https://github.com/my/repo/pull/9128` to prevent the line break?